sotodlib/site_pipeline/make_book.py

```diff
@@ -0,0 +1,52 @@
+import os, os.path as op
+import argparse
+from sotodlib.iobookbinder import Bookbinder
```

Comment:
This line has a typo.  Has this been tested?

Reply:
ah sorry I didn't test the executable script because I'm intending to rewrite it in a new PR, but I've fixed the typo and import now.